### DISCUSSION TITLE
Open Source Promotion Plan Summer 2022

### DISCUSSION THREAD

**Discussion by SkyeYoung** [apache/apisix#7033] [General]
Open Source Promotion Plan is an organized summer program supported by Open Source Software Supply Chain Promotion Plan. It aims to actively encourage college students to participate in the development and maintenance of open-source software actively and promote the vigorous development of an excellent open-source software community.

Open Source Promotion Plan unites major open source communities worldwide to provide project tasks for the development and maintenance of open-source software. In addition, the program is available to university students from all over the world. Students can apply for the project they are interested in and guided by the senior maintainer (community mentor) once they are selected. After completing the project and contributing the results to the community, students will also receive a bonus and certificate for meeting the Open Source Promotion Plan.

The Apache APISIX community has recommended several selected project topics to the Open Source Summer 2022 organizing committee, and the first batch is now online. Welcome to apply!

For the full content you can check [OSPP 2022](https://github.com/apache/apisix/wiki/Open-Source-Promotion-Plan-Summer-2022) or [OSPP 2022 中文版](https://github.com/apache/apisix/wiki/Open-Source-Promotion-Plan-Summer-2022-(Chinese)).

**Comment by reeseliu715** [comment]
There are many challenges as a student at OSPP, but it is a great opportunity for growth.
